src/utils/RouteUtils.js

```diff
@@ -222,7 +223,33 @@ async function parseRoute(resp: Object, destinationName: string) {
                     try {
                         // TODO refactor
                         // eslint-disable-next-line no-await-in-loop
-                        const snappingResponse = await axios.post(`http://${process.env.MAP_MATCHING || 'ERROR'}:8989/match`, gpx, config);
+
+                        const options = {
+                            method: 'POST',
+                            url: `http://${process.env.MAP_MATCHING || 'ERROR'}:8989/match`,
+                            body: gpx,
+                            headers:
```

Comment:
can we just have:
`headers: { ... },`

I don't see the point of having it on 3 lines